File: exercicios em Python/q4.py
import os

print("\nEste programa converte o real em uma de 4 moedas diferentes!")
print("\nmenu")
print("\n1.Dolar.")
print("\n2.Euro")
print("\n3.Libras Esterlinas.")
print("\n4.Renminbi.")
op = int(input("\nDigite a opçao desejada: "))

if op == 1:
    print("\nVoce escolheu o dolar!")
    real = float(input("\nDigite o valor em reais a qual voce deseja converter: "))
    conversao = float(real / 3.90)
    print("\n",real," reais vale ",conversao,"Dolares.")


elif op == 2:
    print("\nVoce escolheu o Euro!")
    real = float(input("\nDigite o valor em reais a qual voce deseja converter: "))
    conversao = float(real / 4.38)
    print("\n",real," reais vale ",conversao,"Euros.")

elif op == 3:
    print("\nVoce escolheu as Libras Esterlinas!")
    real = float(input("\nDigite o valor em reais a qual voce deseja converter: "))
    conversao = float(real / 5.10)
    print("\n",real," reais vale ",conversao,"Libras esterlinas.")

elif op == 4:
    print("\nVoce escolheu o Renminbi!")
    real = float(input("\nDigite o valor em reais a qual voce deseja converter: "))
    conversao = float(real / 0.58)
    print("\n",real," reais vale ",conversao,"Renminbis.")

print("\nObrigado por utilizar nosso sistema!")
os.system ("pause")

File: exercicios em C/q4.cpp
#include <stdio.h>

int main (void){
	int op, real, conversao;
	printf ("\nEste programa converte uma de quatro diferetes moedas em real, com base na cotacao de 28 de marco de 2019.");
	printf ("\nEscolha uma das segintes moedas.");
	printf ("\nMenu: ");
	printf ("\n1 Dolar.");
	printf ("\n2 Libras esterlinas.");
	printf ("\n3 Renminbi.");
	printf ("\n4 Euro.");
	printf ("\nDigite o numero da opcao desejada: ");
	scanf ("%d", &op);
	switch (op){
		case 1:
		printf ("\n\n\t\tDolar.");
		printf ("\nDigite o valor em reais a qual voce quer converter em dolar: ");
		scanf ("%d", &real);
		conversao = real / 3,90;
		printf ("\nO resultado da sua conversao e: %d", conversao);
		break;
			
		case 2:
		printf ("\n\n\t\tLibras Esterlinas.");
		printf ("\nDigite o valor em reais a qual voce quer converter em Libras Esterlinas: ");
		scanf ("%d", &real);
		conversao = real / 5,10;
		printf ("\nO resultado da sua conversao e: %d", conversao);
		break;
			
		case 3:
		printf ("\n\n\t\tRenminbi.");
		printf ("\nDigite o valor em reais a qual voce quer converter em Renminbi: ");
		scanf ("%d", &real);
		conversao = (real * 100) / 58;
		printf ("\nO resultado da sua conversao e: %d", conversao);
		break;
			
		case 4:
		printf ("\n\nvoce escolheu a moeda Euro.");
		printf ("\nDigite o valor em reais a qual voce quer converter em Euro: ");
		scanf ("%d", &real);
		conversao = real / 4,38;
		printf ("\nO resultado da sua conversao e: %d\n", conversao);
		break;
			
		default:
		printf ("\n\nErro! tente novamente, veja se voce digitou uma das opcoes validas.");
		break;
	}
	printf ("\n\nObrigado por utiliza nosso sistema.");
	return 0;
}

